### ISSUE TITLE
Archive explorer ToDos and potential enhancements

### ISSUE THREAD

**Issue by nevrome** [poseidon-framework/poseidon-framework.github.io#52]
I just published the archive explorer. Here are some ToDos for the future, ideally for @93Boy. Generally each of them deserves an own Pull Request, but maybe the map-related ones can be solved in one go.

- [ ] Write a little blog post at https://blog.poseidon-adna.org about the new archive explorer to introduce it to the world.
***
Currently possible changes for the map:
- [ ] The per-sample popups in the map should only show the information we actually have for a given sample, so the empty fields should be omitted.
- [ ] The sample popups should also have a button to open the relevant per-package page.
- [ ] Modern and ancient samples should be distinguished by marker shape in the map (although the marker clusters generally omit the individual markers, admittedly...)
***
Currently possible changes beyond the map:
- [ ] The per-package pages should include some more summary statistics for the package. E.g. How many modern/ancient samples are in the package? How many are male/female/unknown? What is the mean number of SNPs?
- [ ] We should get a clever mechanism to attribute each per-package page an own URL, so that they can be referenced directly.
- [ ] List old package versions on the per-package pages.
***
Changes that will be possible in the future with a more beefed-up version of the Web API:
- [ ] Load detailed sample information for the per-package pages only per package to safe bandwidth.
- [ ] Display more general meta/context information (e.g. contributors) for each package on the per-package pages.
- [ ] Render the package changelog on the per-package pages as we did for the old homepage.
- [ ] Render the literature references on the per-package pages as we did for the old homepage.

@stschiff Do you agree with this? Feel free to add or change any of these suggestions.

**Comment by stschiff** [comment]
Yes, perfect summary. Thanks! I think @93Boy / @dhananjaya93 can start on this any time.

**Comment by nevrome** [comment]
This issue is technically obsolete with the trident-provided html API. But above are still some ideas that could eventually be picked up.

The following issue documents ToDos for the new archive explorer: https://github.com/poseidon-framework/poseidon-hs/issues/333
